packages/web-components/src/components/va-combo-box/test/va-combo-box.e2e.ts

```diff
@@ -0,0 +1,159 @@
+    expect(combobox).toEqualAttribute('data-default-value', 'bar');
+  });
+
+  it('renders error message', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(`
+          <va-combo-box label="A label" error="test error message">
+            <option value="foo">Foo</option>
+            <option value="bar">Bar</option>
+          </va-select>
+        `);
+    await page.find('va-combo-box >>> input');
+    const element = await page.find('va-combo-box >>>  #input-error-message');
+    expect(element).toEqualHtml(`<span id="input-error-message" role="alert">
+            <span class="usa-sr-only">Error</span>
+            <span class="usa-error-message">test error message</span>
+            </span>`);
+  });
+
+  it('renders label with required', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(`
+          <va-combo-box label="A label" required>
+            <option value="foo">Foo</option>
+            <option value="bar">Bar</option>
+          </va-select>
+        `);
+    await page.find('va-combo-box >>> input');
+    const element = await page.find('va-combo-box >>> label');
+    expect(element)
+      .toEqualHtml(`<label class="usa-label" for="options" id="options-label">
+            A label
+            <span class="usa-label--required">  (*Required)</span>
+            </label>`);
+  });
+
+  it('renders placeholder text ', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(`
+          <va-combo-box label="A label" placeholder="test placeholder">
+            <option value="foo">Foo</option>
+            <option value="bar">Bar</option>
+          </va-select>
+        `);
+    await page.find('va-combo-box >>> input');
+    const element = await page.find('va-combo-box >>> input');
+    expect(element).toEqualAttribute('placeholder', 'test placeholder');
+  });
+
+  it('renders hint text ', async () => {
+    const page = await newE2EPage();
+    const hintText = 'test hint';
+    const labelText = 'A label';
+
+    await page.setContent(`
+          <va-combo-box label="${labelText}" hint="${hintText}">
+            <option value="foo">Foo</option>
+            <option value="bar">Bar</option>
+          </va-select>
+        `);
+    await page.find('va-combo-box >>> input');
+
+    const hintElement = await page.find('va-combo-box >>> #input-hint');
+    expect(hintElement).toEqualText('test hint');
+    expect(hintElement).toEqualHtml(`<span class="usa-hint" id="input-hint">
+            ${hintText}
+            </span>`);
+    const element = await page.find('va-combo-box');
+
+    // validate that the label is followed by the hint
+    expect(element.shadowRoot.innerHTML).toContain(
+      `<label for="options" class="usa-label" id="options-label">${labelText}</label><span class="usa-hint" id="input-hint">${hintText}</span>`,
+    );
+  });
+
+  it('renders disabled', async () => {
```

Comment:
At a minimum we'd like to see a test for every prop and event for the component. That means I'd expect a minimum of 13 tests here but I only see 7. Do you mind beefing up these tests a little bit more?

Reply:
sure, I can work on adding more coverage here